lib/internal/streams/operators.js

```diff
@@ -229,9 +229,40 @@ async function toArray(options) {
   }
   return result;
 }
+
+async function* flatMap(fn, options) {
+  for await (const val of this.map(fn, options)) {
+    yield* val;
+  }
+}
+
+async function* drop(number) {
+  validateInteger(number, 'number', 0);
+  for await (const val of this) {
+    if (number-- <= 0) {
+      yield await val;
```

Comment:
Why `yield await val`, while `take()` below does `yield val`?

Reply:
@vweevers I had a bug in the test and `yield await` alters the stack trace in V8 to show another frame. I'll remove it. Thanks.